src/_guides/language/language-tour.md

```diff
@@ -3547,6 +3554,66 @@ enum Color { red, green, blue }
   to help prevent copy-paste errors.
 {{site.alert.end}}
 
+#### Declaring enhanced enums
+
+Dart also allows enum declarations to declare classes
+with fields, methods, and const constructors
+which are limited to a fixed number of known constant instances.
+
+To declare an enhanced enum,
+follow a syntax similar to normal [classes](#classes),
+but with a few extra requirements:
+
+* Instance variables must be `final`, 
+  including those added by [mixins](#mixins).
+* All [generative constructors](#constant-constructors) must be constant.
+* [Factory constructors](#factory-constructors) can only return
+  one of the fixed, known enum instances.
+* No other class can be extended, [`Enum`] is automatically extended.
```

Comment:
This line feels contradictory. Suggestion: replace the comma with as.

Reply:
```suggestion
* No other class can be extended as [`Enum`] is automatically extended.
```